src/placement/jump_map.c

```diff
@@ -33,16 +33,35 @@
 
 
 /*
- * These ops determine whether extra information is calculated during
- * placement.
+ * These operations determine whether extra information is calculated during
+ * placement. i.e. whether the layout is extended or extra addition calculations
+ * are performed.
  *
- * PL_PLACE_EXTENDED calculates an extended layout for use when there
- * is a reintegration operation currently ongoing.
+ * PL_PLACE		This operation will generate a layout including spare
+ * 			targets in the event some of the original targets are
+ * 			unavailable.
  *
- * PL_REINT calculates the post-reintegration layout for use during
- * reintegration, it treats the UP status targets as UP_IN.
+ * PL_PLACE_EXTENDED	This operation will produce a layout that will include
+ * 			extra targets within the layout. These targets are
+ * 			needed to facilitate online operations so that new data
+ * 			will be written to the new and current layouts equally.
+ * 			This is important in case an operation fails so that
+ * 			the old targets aren't in a stale state.
  *
- * Currently the other OP types calculate a normal layout without extra info.
+ * PL_REBUILD		Currently a place-holder. This functions the same as
+ * 			PL_PLACE.
+ *
+ * PL_REINT		This operation will calculate placement for the
+ * 			reintegration operation. It will keep track of any
+ * 			targets to be reintegrated during the initial placement
+ * 			calculation and will then extend the layout, including
+ * 			them before returning.
+ *
+ * PL_ADD		This operation will generate an extended layout
+ * 			including the newly added storage targets. By using this
+ * 			operation the placement calculation will consider newly
+ * 			added targets in the Jump Consistent Hashing calculation
+ * 			where it would normally not.
```

Comment:
```suggestion
 *			targets in the event some of the original targets are
 *			unavailable.
 *
 * PL_PLACE_EXTENDED	This operation will produce a layout that will include
 *			extra targets within the layout. These targets are
 *			needed to facilitate online operations so that new data
 *			will be written to the new and current layouts equally.
 *			This is important in case an operation fails so that
 *			the old targets aren't in a stale state.
 *
 * PL_REBUILD		Currently a place-holder. This functions the same as
 *			PL_PLACE.
 *
 * PL_REINT		This operation will calculate placement for the
 *			reintegration operation. It will keep track of any
 *			targets to be reintegrated during the initial placement
 *			calculation and will then extend the layout, including
 *			them before returning.
 *
 * PL_ADD		This operation will generate an extended layout
 *			including the newly added storage targets. By using this
 *			operation the placement calculation will consider newly
 *			added targets in the Jump Consistent Hashing calculation
 *			where it would normally not.
```